core/src/main/java/io/confluent/rest/exceptions/DebuggableExceptionMapper.java

```diff
@@ -59,11 +59,11 @@ public DebuggableExceptionMapper(RestConfig restConfig) {
       readableMessage += " " + exc.getClass().getName() + ": " + exc.getMessage();
       try {
         ByteArrayOutputStream os = new ByteArrayOutputStream();
-        PrintStream stream = new PrintStream(os);
+        PrintStream stream = new PrintStream(os, false, "UTF-8");
```

Comment:
I'm not that picky about it, but some people might prefer using `StandardCharsets.UTF_8.name()` to avoid repeating this string constant and making any issues w/ it a compile-time failure.